samples/net/sockets/echo_server/overlay-tfm.conf

```diff
@@ -3,8 +3,5 @@ CONFIG_BUILD_WITH_TFM=y
 CONFIG_NORDIC_SECURITY_BACKEND=y
 CONFIG_MBEDTLS_PSA_CRYPTO_C=y
 
-CONFIG_MBEDTLS=y
-CONFIG_MBEDTLS_LIBRARY=y
-CONFIG_MBEDTLS_INSTALL_PATH="DUMMY"
```

Comment:
This comment belongs to the https://github.com/nrfconnect/sdk-zephyr/pull/525/commits/6988676aab993be50c15d0184cf6aecbf7f2c897 commit message, and is placed here because that commit changes this line. But nothing is wrong with the code.

I think this should be a temphack, because it follows this commit: https://github.com/frkv/fw-nrfconnect-zephyr/commit/c5feda0deb63888d33331874b24c69308b11b824 which is already a temphack.

Could you maybe make a comment in the commit message, that this commit should be squashed with the existing https://github.com/frkv/fw-nrfconnect-zephyr/commit/c5feda0deb63888d33331874b24c69308b11b824 commit on the next sdk-zephyr patch clean-up / history re-write.